src/catalog/pricing-advanced.md

```diff
@@ -67,7 +67,8 @@ Sets up promotional and tier prices for specific shared catalogs and customer gr
 ![]({% link images/images-b2b/product-pricing-advanced.png %}){: .zoom}
 _Advanced Pricing_{:.b2b-only}
 
-|Website|Identifies the website where the group price rule applies. This option appears only if the installation has multiple websites.|
+|Website|Identifies the website where the group price rule applies. This option appears only if the installation has multiple websites.
+**Important**: the _Website_ value should also be selected in the [Catalog Price Scope] ({% link catalog/catalog-price-scope.md %}) configuration, otherwise the set advanced prices will be displayed for **all** websites.|
```

Comment:
```suggestion
**_Important:_** The _Website_ value should also be selected in the [Catalog Price Scope] ({% link catalog/catalog-price-scope.md %}) configuration, otherwise the set advanced prices will be displayed for **all** websites.|
```

Reply:
@leslietilling thank you for reviewing ☺️

Your suggestion is done - please approve.